scripts/requirements.txt

```diff
@@ -1,3 +1,4 @@
+coremltools
```

Comment:
I think we are trying not to duplicate requirements between `setup.py` and `requirements.txt` -- and coremltools is already in setup.py. Why is it needed here? (If all setup.py requirements are needed here, we should copy them all over.)

Reply:
All the requirements from `setup.py` except this one are already in this file.

Reply:
I see. In that case, should we pin a version here so it doesn't drift from the setup.py version?